content/zh-cn/blog/_posts/2022-08-31-cgroupv2-ga.md

```diff
@@ -0,0 +1,305 @@
+management capabilities.
+-->
+Kubernetes 1.25 将 cgroup v2 升级为一般可用性（GA），
+让 [kubelet](/zh-cn/docs/concepts/overview/components/#kubelet) 使用最新的容器资源管理能力。
+
+<!--
+## What are cgroups?
+-->
+## 什么是 cgroup？
+
+<!--
+Effective [resource management](/docs/concepts/configuration/manage-resources-containers/) is a
+critical aspect of Kubernetes. This involves managing the finite resources in
+your nodes, such as CPU, memory, and storage.
+
+*cgroups* are a Linux kernel capability that establish resource management
+functionality like limiting CPU usage or setting memory limits for running
+processes.
+-->
+有效的[资源管理](/zh-cn/docs/concepts/configuration/manage-resources-containers/)是 Kubernetes 的一个关键方面。
+这涉及管理节点中的有限资源，例如 CPU、内存和存储。
+
+*cgroups* 是一种 Linux 内核功能，可建立资源管理功能，
+例如限制 CPU 使用率或为正在运行的进程设置内存限制。
+
+<!--
+When you use the resource management capabilities in Kubernetes, such as configuring
+[requests and limits for Pods and containers](/docs/concepts/configuration/manage-resources-containers/#requests-and-limits),
+Kubernetes uses cgroups to enforce your resource requests and limits.
+
+The Linux kernel offers two versions of cgroups: cgroup v1 and cgroup v2.
+-->
+当你使用 Kubernetes 中的资源管理功能时，例如配置 
+[Pod 和容器的请求和限制](/zh-cn/docs/concepts/configuration/manage-resources-containers/#requests-and-limits)，
+Kubernetes 会使用 cgroups 来强制执行你的资源请求和限制。
+
+Linux 内核提供了两个版本的 cgroup：cgroup v1 和 cgroup v2。
+
+<!--
+## What is cgroup v2?
+-->
+## 什么是 cgroup v2？
+
+<!--
+cgroup v2 is the latest version of the Linux cgroup API. cgroup v2 provides a
+unified control system with enhanced resource management capabilities.
+
+cgroup v2 has been in development in the Linux Kernel since 2016 and in recent
+years has matured across the container ecosystem. With Kubernetes 1.25, cgroup
+v2 support has graduated to general availability.
+-->
+cgroup v2 是 Linux cgroup API 的最新版本,
+提供了一个具有增强的资源管理能力的统一控制系统。
+
+自 2016 年以来，cgroup v2 一直在 Linux 内核中进行开发，
+近年来在整个容器生态系统中已经成熟。 在 Kubernetes 1.25 中，
+对 cgroup v2 的支持已升级为一般可用。
+
+<!--
+Many recent releases of Linux distributions have switched over to cgroup v2 by
+default so it's important that Kubernetes continues to work well on these new
+updated distros.
+
+cgroup v2 offers several improvements over cgroup v1, such as the following:
+-->
+默认情况下，许多最新版本的 Linux 发行版已切换到 cgroup v2，
+因此 Kubernetes 继续在这些新更新的发行版上正常运行非常重要。
+
+cgroup v2 对 cgroup v1 进行了多项改进，例如：
+
+<!--
+* Single unified hierarchy design in API
+* Safer sub-tree delegation to containers
+* Newer features like [Pressure Stall Information](https://www.kernel.org/doc/html/latest/accounting/psi.html)
+* Enhanced resource allocation management and isolation across multiple resources
+    * Unified accounting for different types of memory allocations (network and kernel memory, etc)
+    * Accounting for non-immediate resource changes such as page cache write backs
+-->
+* API 中的单一统一层次结构设计
+* 更安全的子树委派给容器
+* [压力失速信息](https://www.kernel.org/doc/html/latest/accounting/psi.html)等新功能
```

Comment:
```suggestion
* [压力阻塞信息](https://www.kernel.org/doc/html/latest/accounting/psi.html)等新功能
```
Refer to [cgroups](https://kubernetes.io/zh-cn/docs/concepts/architecture/cgroups/#cgroup-v2) for this para.